setup.py

```diff
@@ -12,7 +12,7 @@
 DESCRIPTION = u'Write Start/Stop events in your SQL logs.'
 PACKAGES = ['django_sql_log']
 REQUIREMENTS = [
-    'django<1.7',
+    'Django',
```

Comment:
Perhaps Django was explicitely `<1.7` before, because it was a known bad version. And perhaps 1.9 will be supported without efforts.
At the moment, we don't know, so we may not consider Django 1.9 as a known bad version yet. Ok to keep Django version unlimited.

Just be aware that with unlimited Django version, if next release of Django breaks, then there would be no error in Travis (because it installs Django up to 1.8), but possible errors for users who upgrade to Django 1.9 (and bug reports I guess).